app/assets/javascripts/document.service.js.erb

```diff
@@ -33,11 +33,6 @@ function DocumentService($http, $rootScope){
 			console.log(error);
 		})
 	}
-
-	doc_service.setCurrentLocation = function(lat, lng){
```

Comment:
this wasn't used anywhere.

Reply:
hmm.. i am surprised i didnt notice that